### ISSUE TITLE
Remove Autoload components

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-packages/permut#9]
autoloading of package and its documentation is an obsolete concept, not in use any more. You can remove it from

https://github.com/gap-packages/permut/blob/33a7318d1e1b02ff1358b277eb16796dd7f010ab/PackageInfo.g#L97

and 

https://github.com/gap-packages/permut/blob/33a7318d1e1b02ff1358b277eb16796dd7f010ab/PackageInfo.g#L113

**Comment by RamonEstebanRomero** [comment]
Done. Thank you, @alex-konovalov

**Comment by olexandr-konovalov** [comment]
Thanks @RamonEstebanRomero 

BTW, if you would have typed in the commit message in https://github.com/gap-packages/permut/commit/358976915fb56b558ecbedf7041b5d7aa7e42f54 

```
Update PackageInfo.g

Modified TestFile, removed Autoload components. Closes #9
```

Then this issue would be closed automatically (and also there will be a link between issue and commit in which it has been fixed).

**Comment by olexandr-konovalov** [comment]
A better commit message would be actually
```
Modified TestFile, removed Autoload components

Closes #9.
```

because the fact that `PackageInfo.g` was updated is known from the history anyway (gIthub web interface simply suggested that by default, I know), and there are various shortened views of revisions history where you want this 1st line to be short but informative. This is a classical reference for commits: https://chris.beams.io/posts/git-commit/
